app/Http/Controllers/CustomFieldsetsController.php

```diff
@@ -187,6 +187,7 @@ public function associate(Request $request, $id)
 
         $results = $set->fields()->attach(Input::get('field_id'), ["required" => ($request->input('required') == "on"),"order" => $request->input('order', 1)]);
 
+
```

Comment:
Can you remove this extra line break? I know it seems petty, but it can add a bit of extra noise to PRs, so it's a good habit to avoid adding extra line breaks and spaces unless they're actually needed for legibility.

Reply:
deleted, ill be more careful reviewing my changes before requesting a PR moving forward

Reply:
Hm, I'm still not seeing the order code in this PR though. If you look at files changed, it only shows the department update stuff and removing the `$maxId` from the blade. Is the `$maxId` the thing you changed to resolve the issue? If so, I'm not sure how that fixes it, since we're not sending a value through there anymore. Probably better to set a default in the controller, no?